templates/catalog/category_tags_filter.html

```diff
@@ -7,12 +7,16 @@
       </p>
 
       <div class="tags-filter-inputs js-tags-inputs">
-        {% for tag in tags %}
-          <p class="checkbox">
-            <input type="checkbox" id="tag-{{ tag.slug }}" data-tag-id="{{ tag.id }}" data-tag-slug="{{ tag.slug }}" data-tag-group-id="{{ tag.group.id }}">
-            <label for="tag-{{ tag.slug }}">{{ tag.name }}</label>
-          </p>
-        {% endfor %}
+        {% if tags|length < 21 %}
+          {% for tag in tags %}
+            <p class="checkbox">
+              <input type="checkbox" id="tag-{{ tag.slug }}" data-tag-id="{{ tag.id }}" data-tag-slug="{{ tag.slug }}" data-tag-group-id="{{ tag.group.id }}">
+              <label for="tag-{{ tag.slug }}">{{ tag.name }}</label>
+            </p>
+          {% endfor %}
+        {% else %}
+        &nbsp от {{ tags|first }} до {{ tags|last }}
```

Comment:
How exactly will this work? Will be it slider or smth else?
Can we just add a “Show all” button for a group in which the number of tags is greater than 21? How it is done here:
<img width="167" alt="default" src="https://user-images.githubusercontent.com/13870342/49981283-35420180-ff68-11e8-8b63-bc2a92a1a0a8.png">

It will be simpler

Reply:
@artemiy312 , sorry, i forgot to pin screenshot.
I'll describe in task comment how this should work and create subtask here

Reply:
@artemiy312 , it's UI design decision, so it's not correct to discuss it in PR.
See this comment in related task: https://github.com/fidals/stroyprombeton/issues/374#issuecomment-447556336

We can continue discussion there